src/forms/inputs/textarea.js

```diff
@@ -36,21 +38,29 @@ import { compose, filterInvalidDOMProps, generateInputErrorId } from '../../util
 const propTypes = {
   ...fieldPropTypes,
   hideCharacterCount: PropTypes.bool,
-  maxLength: PropTypes.oneOfType([PropTypes.number, PropTypes.bool])
+  maxLength: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
+  forwardedRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({
+      current: PropTypes.instanceOf(Element), // eslint-disable-line no-undef
+    }),
+  ]),
 }
 
 const defaultProps = {
   maxLength: null,
   hideCharacterCount: false,
+  forwardedRef: null,
 }
 
-function Textarea (props) {
+const Textarea = forwardRef(function Textarea (props, ref) {
```

Comment:
Can you include a screenshot of how this component displays in React dev tools? This approach is new to me and I want to make sure that the advice from that GitHub issue works!

Reply:
Here it is after rendering it with a standard `ref` passed in: https://share.getcloudapp.com/YEuXONjN

Reply:
🙌  that looks great